verisignmdns/src/main/java/denominator/verisignmdns/VrsnMdnsContentHandler.java

```diff
@@ -0,0 +1,167 @@
+package denominator.verisignmdns;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Inject;
+
+import org.xml.sax.Attributes;
+import org.xml.sax.SAXException;
+import org.xml.sax.helpers.DefaultHandler;
+
+import denominator.model.Zone;
+import denominator.verisignmdns.VrsnMdns.Record;
+import feign.sax.SAXDecoder.ContentHandlerWithResult;
+
+class VrsnMdnsContentHandler {
+
+    static class ZoneListHandler extends DefaultHandler implements ContentHandlerWithResult<List<Zone>> {
+        @Inject
+        ZoneListHandler() {
+        }
+
+        private final List<Zone> zones = new ArrayList<Zone>();
+        private boolean domainElementFound = false; // flag for getting the
+                                                    // value...
+
+        @Override
+        public List<Zone> result() {
+            return zones;
+        }
+
+        @Override
+        public void startElement(String uri, String localName, String qName, Attributes attrs) {
+
+            if (qName != null && qName.endsWith(":domainName")) {
+                domainElementFound = true;
+            }
+        }
+
+        @Override
+        public void characters(char ch[], int start, int length) throws SAXException {
+            if (domainElementFound) {
+                zones.add(Zone.create(new String(ch, start, length)));
+            }
+        }
+
+        @Override
+        public void endElement(String uri, String localName, String qName) throws SAXException {
+            if (qName != null && qName.endsWith(":domainName")) {
+                domainElementFound = false;
+            }
+        }
+    }
+
+    static class RecordListHandler extends DefaultHandler implements ContentHandlerWithResult<List<Record>> {
+        private final List<Record> rrs = new ArrayList<Record>();
+
+        @Inject
+        RecordListHandler() {
+        }
+
+        private Record rr = new Record();
+        private boolean processingRR = false; // flag indicating currently
```

Comment:
using descriptive field names is better than 3 line comments. For example, `inResourceRecord` explains this.

ex. https://github.com/Netflix/denominator/blob/master/route53/src/main/java/denominator/route53/ListResourceRecordSetsResponseHandler.java#L29